vnext/src/IntegrationTests/AsyncStorageTest.windesktop.js

```diff
@@ -43,15 +44,32 @@ function expectTrue(condition: boolean, message: string) {
   }
 }
 
+// Type-safe wrapper around JSON.stringify
+function stringify(
+  value:
+    | void
+    | null
```

Comment:
does 'flow' duplicate null and undefined, or does it implicitly allow undefined ? 
Recent updates for TypeScript, C# have moved to explicitly naming undefined/null in the type system to have the compiler error earlier if something can be null.
I know we have the type case below. but in TypeScript noone can call this function with undefined as undefined is not a legal type for value...

Reply:
I don't know, but your reasoning suggests flow does allow it.

Note, this is a forked file with contents almost identical from the RN original, so even questionable code is meant to stay as is.